tools/legal-review/engine/io.helidon.common.helidon-common-4.0.8/copyright-ignore

```diff
@@ -0,0 +1,14 @@
+Copyright (c) 2017, 2022 Oracle and/or its affiliates.
+Copyright (c) 2017, 2023 Oracle and/or its affiliates.
+Copyright (c) 2018, 2021 Oracle and/or its affiliates.
+Copyright (c) 2018, 2024 Oracle and/or its affiliates.
+Copyright (c) 2019, 2020 Oracle and/or its affiliates.
+Copyright (c) 2019, 2021 Oracle and/or its affiliates.
+Copyright (c) 2019, 2022 Oracle and/or its affiliates.
+Copyright (c) 2019, 2023 Oracle and/or its affiliates.
+Copyright (c) 2020 Oracle and/or its affiliates.
+Copyright (c) 2021 Oracle and/or its affiliates.
+Copyright (c) 2021, 2023 Oracle and/or its affiliates.
+Copyright (c) 2022 Oracle and/or its affiliates.
+Copyright (c) 2023 Oracle and/or its affiliates.
+Copyright (c) 2024 Oracle and/or its affiliates.
```

Comment:
We should probably not be ignoring almost all of these - I guess at least to cover the whole timespan these are representing - e.g. 2017-2024.

I think it's safest to do "when in doubt, prefer to keep".

Reply:
updated :+1: